README.md

```diff
@@ -1,5 +1,4 @@
-`xcaddy` - Custom Caddy Builder
-===============================
+# `xcaddy` - Custom Caddy Builder
```

Comment:
This change seems unnecessary

Reply:
The document was formmated by prettier. I'll remove the unnecessary changes.